#!/usr/bin/env python
PACKAGE = "uchile_laser_pipeline"

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, double_t, bool_t

gen = ParameterGenerator()

gen.add("l_radius", double_t, 0, "Left hand inflation radius. Only valid if greater than the hand radius.", 0.1, 0.0, 0.5)
gen.add("r_radius", double_t, 0, "Right hand inflation radius. Only valid if greater than the hand radius.", 0.1, 0.0, 0.5)
gen.add("publish_markers", bool_t, 0, "Whether the filter publishes rviz markers or no.", True)

exit(gen.generate(PACKAGE, "uchile_laser_pipeline", "SelfFilter"))
